atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -73,6 +73,8 @@
                             // CipherSuite.TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256,
                             // CipherSuite.TLS_RSA_WITH_AES_256_GCM_SHA384,
                             // CipherSuite.TLS_RSA_WITH_AES_128_GCM_SHA256,
+                            CipherSuite.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
```

Comment:
We should add a comment here referencing https://http2.github.io/http2-spec/index.html#rfc.section.9.2.2

> The black list includes the cipher suite that TLS 1.2 makes mandatory, which means that TLS 1.2 deployments could have non-intersecting sets of permitted cipher suites. To avoid this problem causing TLS handshake failures, deployments of HTTP/2 that use TLS 1.2 MUST support TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 [TLS-ECDHE] with the P-256 elliptic curve [FIPS186].

I know we're also gathering some perf comparisons of throughput and latency, would like to include as much detail in the PR here as possible.